tests/integration/targets/nomad/tasks/main.yml

```
@@ -0,0 +1,68 @@
+- name: Install Nomad and test
+  vars:
+    nomad_version: 0.12.3
+    nomad_uri: https://releases.hashicorp.com/nomad/0.12.3/nomad_{{ nomad_version }}_{{ ansible_system | lower }}_{{ nomad_arch }}.zip
+    nomad_cmd: '{{ output_dir }}/nomad'
+  block:
+  - name: register pyOpenSSL version
+    command: '{{ ansible_python_interpreter }} -c ''import OpenSSL; print(OpenSSL.__version__)'''
+    register: pyopenssl_version
+  - name: Install requests<2.20 (CentOS/RHEL 6)
```

```suggestion
    nomad_cmd: '{{ output_dir }}/nomad'
    
  block:
  - name: register pyOpenSSL version
    command: '{{ ansible_python_interpreter }} -c ''import OpenSSL; print(OpenSSL.__version__)'''
    register: pyopenssl_version

    - name: Install requests<2.20 (CentOS/RHEL 6)
```
could you please put blank lines between tests? it'll improve readability

---

ok